util/src/main/java/tech/pegasys/artemis/util/config/ArtemisConfiguration.java

```diff
@@ -399,4 +431,25 @@ public int getBeaconRestAPIPortNumber() {
   public boolean getBeaconRestAPIEnableSwagger() {
     return config.getBoolean("beaconrestapi.enableSwagger");
   }
+
+  private LoggingDestination getLoggingDestination() {
```

Comment:
Typically `ArtemisConfiguration` sticks to just the raw data types so I think we probably should invert this dependency.  So `ArtemisConfiguration.getLoggingDestination` just returns a string and `LoggingDestination` has a `fromConfig` that has this mapping.